zh/monitor-using-tidbmonitor.md

```diff
@@ -39,12 +39,12 @@ spec:
   - name: basic
   prometheus:
     baseImage: prom/prometheus
-    version: v2.11.1
+    version: latest
```

Comment:
Please change the version of Prometheus to `v2.18.1` and version of Grafana to `6.1.6`, we do not use the latest version.

Reply:
fixed